### ISSUE TITLE
Cannot find module 'browserify-shim'

### ISSUE THREAD

**Issue by vpowers** [videojs/mux.js#131]
I'm using [video-contrib-hls](https://github.com/videojs/videojs-contrib-hls) in a project that uses browserify to bundle via gulp.  I'm getting a compile error:

Error: Cannot find module 'browserify-shim' from '/node_modules/mux.js'

Any idea why this module wouldn't be installed as a dependency when using video-contrib-hls?

**Comment by cytoplankton** [comment]
I'm also getting this error when trying to use [video-contrib-hls](https://github.com/videojs/videojs-contrib-hls). I'm able to get videojs loading just fine using es6 syntax, but not video-contrib-hls.

I've tried both this
```
import videojs from 'video.js';
require('videojs-contrib-hls');
```
and this
```
import videojs from 'video.js';
import 'videojs-contrib-hls';
```
Thanks in advance!

**Comment by fozzle** [comment]
Having the same issue.

**Comment by ScottLNorvell** [comment]
Same issue.

I can trace it to this commit https://github.com/videojs/mux.js/commit/c92e0dcbc6732b1095e209e2f7af3b03dd918421

from the pr https://github.com/videojs/mux.js/pull/127

To solve, I had to add videojs-contrib-media-sources to my package json, lock it to 4.1.0 (in 4.2.0 they bumped their mux dependency). Then I added mux.js locked to 3.0.2 to solve.

Any chance we could move browserify-shim back to devDependencies?

**Comment by gesinger** [comment]
A fix for this was merged in via https://github.com/videojs/mux.js/pull/151 , and was released as part of v4.1.3. videojs-contrib-media-sources and videojs-contrib-hls were also updated to point to the new version. Please give the latest versions a shot and let us know if you are still experiencing issues. Thanks to all for reporting the issue, and for the debugging information.
